install-template.sh

```diff
@@ -174,11 +174,11 @@ valopt() {
         eval $v="$default"
         for arg in $CFG_ARGS
         do
-            if echo "$arg" | grep -q -- "--$op="
-            then
-                local val=$(echo "$arg" | cut -f2 -d=)
-                eval $v=$val
-            fi
+            case "${arg}" in
+                "--${op}="* )
```

Comment:
FYI, this is technically a behavioral change because the original code `grep`ped for `"--$op"` instead of `"^--$op"` -- testing for the prefix is probably intended, though. Same for the `--$option` case below.